src/sass/components/categories.scss

```
@@ -21,15 +25,34 @@
     font-size: 12px;
     line-height: 1.3;
     cursor: pointer;
+
+    &_button:nth-of-type(odd) {
+      background: cadetblue;
+    }
   }
 }
 
-.as {
-  position: relative;
+.special_but{
+position: relative;
+width: 112px;
+
+}
```

Додай, будь ласка, відступи
```suggestion
.special_but {
    position: relative;
    width: 112px;
}
```